.circleci/config.yml

```diff
@@ -0,0 +1,63 @@
+version: 2
+
+defaults: &defaults
+    parallelism: 1
+    working_directory: /tmp/cv-rollbar
+    environment:
+        CIRCLE_ARTIFACTS: /tmp/circleci-artifacts
+        CIRCLE_TEST_REPORTS: /tmp/circleci-test-results
+
+jobs:
+  build:
+    <<: *defaults
+    docker:
+      - image: fpco/stack-build
+    steps:
+    - checkout
+
+    - run:
+        name: Initialize workspace
+        command: mkdir -p workspace
+    - run:
+        name: Initialize artifact storage
+        command: mkdir -p $CIRCLE_ARTIFACTS $CIRCLE_TEST_REPORTS
+
+    - restore_cache:
+        keys:
+        - cv-rollbar-cache-v1-dep-{{ .Branch }}-{{ checksum "rollbar.cabal" }}
+        # This branch if available
+        - cv-rollbar-cache-v1-dep-{{ .Branch }}-
+        # Default branch if not
+        - cv-rollbar-cache-v1-dep-master-
+        # Any branch if there are none on the default branch -
+        # this should be unnecessary if you have your default branch
+        # configured correctly
+        - cv-rollbar-cache-v1-dep-
+    - run:
+        name: Build binary
+        command: stack build --install-ghc --pedantic --ghc-options="-O2"
+
+    # Save dependency cache
+    - save_cache:
+        key: cv-rollbar-cache-v1-dep-{{ .Branch }}-{{ checksum "rollbar.cabal" }}
```

Comment:
Now that the cabal file doesn't include version bounds this needs to include the `stack.yaml` checksum as well. Otherwise we'll try to use the same dependencies across different resolvers.

Reply:
Good catch!